test/snippet/io/alignment_file/alignment_file_input_construction_from_stream.cpp

```diff
@@ -0,0 +1,18 @@
+#include <sstream>
+#include <string>
+
+#include <seqan3/io/alignment_file/input.hpp>
+
+int main()
+{
+    std::string input
```

Comment:
why not using a string literal here? Please make it consistent through all snippets.